great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -2662,7 +2662,7 @@ def get_validator(  # noqa: PLR0913
             > 1
         ):
             ge_cloud_mode = getattr(  # attr not on AbstractDataContext
-                self, "ge_cloud_mode"
```

Comment:
This is more thoroughly addressed in this PR
* https://github.com/great-expectations/great_expectations/pull/8433